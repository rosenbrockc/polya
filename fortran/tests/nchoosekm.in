2 2 2
1 2 1
3 4 5
1 1 1
10 0 0
2 3 9
