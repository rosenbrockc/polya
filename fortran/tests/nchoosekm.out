90
12
27720
6
1
20020